src/tutorials/0005/03.md

```diff
@@ -0,0 +1,43 @@
+
+## Working with files in ProtoSchool
+Here in our ProtoSchool tutorials, we create a new IPFS node for you in the browser each time you hit the "Submit" button in a lesson. Whenever you see `ipfs.someMethod()` in our lessons, `ipfs` is a variable that refers to your IPFS instance, also known as a node. The actions that you take only affect your own IPFS node, not nodes belonging to your peers.
```

Comment:
Hmmm, so I have been under the impression that we're creating a new node in the browser each time they hit submit on a lesson and not actually labelling any files that are on their computer, not pinning anything, and neither they nor we are keeping a node live, so they are in effect not putting anything accessible on the network. As in, don't worry too much about which files you upload for these exercises, they aren't ending up on the gateway and no one knows enough to find them. When we give them challenges to access specific files we've made for them, we're also creating them locally in the node we create rather than putting them on the gateway because it was taking impossibly to infinitely long to grab them otherwise. I assume all of this stuff disappears as soon as they close their browser tab? This is the "special"ness of what I thought we were doing; we're making something even less permanent than it usually is. 

Somewhere else there's a note to the effect of "when you're ready to do this thing for real you'll need to learn how to install IPFS and host your stuff"

Was some of this wording  more accurate for the MFS lesson we copied it from? Like making directories and such affected no one but you? 

We can add some disambiguation here about adding being just to your own node but other methods searching your node first, then other nodes if not found there. If so, we could either do it here or wait and do it as we present each method. Or we could mention pinning/sharing here and link to some resource you'd recommend about that.

Reply:
@alanshaw does this edit address your concerns? I propose using this text to clarify what I was trying to express here (the one-ephemeral-node-per-lesson concept) and then later address lesson-by-lesson where stuff goes or doesn't go when you use a specific method.
> Here in our ProtoSchool tutorials, we create a new IPFS node for you in the browser each time you hit the "Submit" button in a lesson. Whenever you see `ipfs.someMethod()` in our lessons, `ipfs` is a variable that refers to your IPFS instance, also known as a node. This IPFS node is not carried over from one lesson to another, which is why you'll see that we often prepopulate some code in an exercise to get your new IPFS node to the state you got a different IPFS node to in a previous one.
> 
> We create these IPFS nodes behind the scenes so that you can focus on the content of each lesson.  Outside of ProtoSchool, though, you'd want a more consistent experience and a node (or multiple nodes) that you could access repeatedly. To acheive this, you could either initialize [JS IPFS in the browser](https://github.com/ipfs/js-ipfs#use-in-the-browser) on your own, or host your own node locally by installing IPFS and running a daemon in your terminal. When you're ready to experiment with that, you can visit the IPFS docs site to learn how to [install IPFS](https://docs.ipfs.io/guides/guides/install/) and [initialize your node](https://docs.ipfs.io/introduction/usage/#initialize-the-repository).

Reply:
Added this ^ plus individual updates to method lesson in my latest commits. Marking resolved but we can address if you think any of my edits don't do the concepts justice.